src/test/java/io/vertx/core/eventbus/ClusteredEventBusTest.java

```diff
@@ -516,4 +516,24 @@ public void init(Vertx vertx, NodeSelector nodeSelector) {
     assertNotNull(nodeSelectorRef.get());
     assertFalse(nodeSelectorRef.get().wantsUpdatesFor(ADDRESS1));
   }
+
+  @Test
+  public void testClusterEventBusClosesItsInternalNetServer() throws Exception {
+    startNodes(2);
+
+    final MessageProducer<String> producer = vertices[0].eventBus().sender(ADDRESS1, new DeliveryOptions().setLocalOnly(false));
+
+    ((VertxInternal) vertices[1]).addCloseHook(
```

Comment:
the lambda argument of `addCloseHook` should be kept in local variable of the `testClusterEventBusClosesItsInternalNetServer` otherwise it might be garbage collected and the test would not be correct